### ISSUE TITLE
San D'oria Mission 2-3 Journey to Bastok (bastok dragon fight)

### ISSUE THREAD

**Issue by Craaizy** [EdenServer/community#556]
I'm not recieving the key item that allows me to start the Rank 2 Final Mission BCNM...i've talked to Pius and Grohm (Grom?) in the metalworks went to the shrine clicked on the Burning circle and no options come up....seems to be the NPCs are not giving me the correct Key Item to start the fight.

**Comment by godmode** [comment]
For any devs picking up this ticket in the future... `!setplayervar {playername} MissionStatus 10` is what is needed here to adjust live. For some reason this is set to 0 somewhere and I'm unable to reproduce it.

**Comment by LunchEden** [comment]
I have a similar issue. I completed all the prereqs to the fight last year back on Kupo. I completed the Windurst segment (trading shields to Halver), started the Bastok segment (Journey to Bastok), and spoke with Savae E Paleade, then Pius, then Grohm. I appear to be getting the correct dialogue from these Npcs (Grohm speaks of investigating a "strange circle"). However when I click on the  burning circle in the Palaborough Mines it does not allow me the option to enter (only option is "None."). I've went back several times and spoke to the appropriate Npcs, relogged, re zoned, and nothing appears to work. It seems I can't advance the mission into the fight and I'm in some sort of weird limbo.
